bootstrap/index.ts

```diff
@@ -33,7 +33,7 @@
         const bundlePath = aliucordDir + "/Aliucord.js.bundle";
         if (!AliuFS.exists(bundlePath)) await download("https://raw.githubusercontent.com/Aliucord/AliucordRN/builds/Aliucord.js.bundle", bundlePath);
 
-        (globalThis._globals ??= {}).aliucord = AliuHermes.run(bundlePath);
+        globalThis.aliucord = AliuHermes.run(bundlePath);
```

Comment:
this is another not mentioned change that would also have to be updated in the rollup plugin. Why was this ever called _globals.aliucord? It seems the purpose is to "hide" it, but being able to access this conveniently is important for debugging things on the console. And if anyone were to use this programatically (why would you care? Let them do it if they really want to), they would still do it regardless of the name. And if this is for "avoiding name clashes", i think it should be obvious that "aliucord" is less likely to clash with something else than "_globals"